server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,76 @@
+[id="proc-creating-oidc-client_{context}"]
+==== Creating an Open ID Connect Client
+To use xref:_oidc[OpenID Connect], an existing application must be secured as an Open ID connect client.
+
+.Procedure
+. Click `Clients` in the left navigation pane.  
++
+.Clients
+image:{project_images}/clients.png[Clients]
+
+. Click *Create* to go to the `Add Client` page.
++
+.Add Client
+image:{project_images}/add-client-oidc.png[Add Client]
+
+. Enter the `Client ID` of the client.
+
+. Select `openid-connect` in the `Client Protocol` drop down box.
+
+. Enter the base URL of your application in the `Root URL` field.
+
+. Click *Save*.  This creates the client and bring you to the `Settings`
```

Comment:
I typically reminder developers to always follow "this" with a noun for clarity about what "this" is. So it would be "this action."